modules/store/src/models.ts

```diff
@@ -39,3 +39,9 @@ export type SelectorWithProps<State, Props, Result> = (
   state: State,
   props: Props
 ) => Result;
+
+export interface RuntimeChecks {
+  strictStateSerializabilityChecks: boolean;
```

Comment:
We are already calling these `RuntimeChecks`, so we can remove `Checks` from the properties themselves.